modules/apps/depot/depot-web/src/main/resources/META-INF/resources/screen/navigation/entries/languages.jsp

```diff
@@ -54,26 +34,9 @@ boolean inheritLocales = GetterUtil.getBoolean(typeSettingsProperties.getPropert
 	<liferay-ui:message key="please-enter-a-valid-name" />
 </liferay-ui:error>
 
-<div class="site-languages">
-
-	<%
-	HashMap<String, Object> data = HashMapBuilder.<String, Object>put(
-		"availableLocales", DepotLanguageUtil.getAvailableLocalesJSONArray(locale)
-	).put(
-		"defaultLocaleId", LocaleUtil.toLanguageId(company.getDefaultUser().getLocale())
-	).put(
-		"inheritLocales", inheritLocales
-	).put(
-		"siteAvailableLocales", DepotLanguageUtil.getDepotAvailableLocalesJSONArray(group, locale)
-	).put(
-		"siteDefaultLocaleId", LocaleUtil.toLanguageId(PortalUtil.getSiteDefaultLocale(group.getGroupId()))
-	).put(
-		"translatedLanguages", group.getNameMap()
-	).build();
-	%>
-
-	<react:component
-		data="<%= data %>"
-		module="js/Languages.es"
-	/>
-</div>
\ No newline at end of file
+<liferay-frontend:languages
+	inheritLocales="<%= inheritLocales %>"
+	siteAvailableLocales="<%= DepotLanguageUtil.getDepotAvailableLocales(group) %>"
+	siteDefaultLocale="<%= PortalUtil.getSiteDefaultLocale(group.getGroupId()) %>"
+	translatedLanguages="<%= group.getNameMap() %>"
```

Comment:
Why are we passing here a map with the translated group names? Shouldn't we pass in the set of locales that have a translation? (e.g. group.getNameMap().keySet()). It feels really weird, because the attribute name implies it expects a bunch of languages, but we're passing a map.